### ISSUE TITLE
blacklist_keys not being filtered out in version 4.2.3

### ISSUE THREAD

**Issue by nbdavies** [airbrake/airbrake-ruby#468]
* airbrake-ruby version 4.2.3
* airbrake version 9.0.2 (also saw this issue with 9.0.0, 8.3.2)
* Ruby version: 2.5.5
* Framework name & version: Rails 5.1.6.2

#### Airbrake config
```ruby
Airbrake.configure do |c|
  c.root_directory = Rails.root
  c.logger = Rails.logger
  c.environment = Rails.env
  c.performance_stats = true
  c.blacklist_keys = Rails.application.config.filter_parameters
end
```

#### Description
The `blacklist_keys` are no longer being filtered out of the parameters. I can test this way from the Rails console:
```ruby
Airbrake.notify("Error Message", { password: "My voice is my passport. Verify me." })
```
On airbrake-ruby version 4.2.2, the parameters show up in the Airbrake UI as:
```
{
  "password": "[Filtered]"
}
```
On version 4.2.3 of the gem, that filtering doesn't happen:
```
{
  "password": "My voice is my passport. Verify me."
}
```

I was able to get blacklist filtering to work again by adding this to the initializer, right after the `Airbrake.configure` block:
```ruby
Airbrake.instance_variable_get(:@notice_notifier).send(:add_default_filters)
```

**Comment by nbdavies** [comment]
It might be related to this change: https://github.com/airbrake/airbrake-ruby/pull/464/files?file-filters%5B%5D=.rb#diff-7826f2319fdc504a7b95dc815db44b04R465

My suspicion is: calling `Airbrake.configure` at the bottom of the file would initialize the notifiers, without the app's specific `blacklist_keys` configuration. Then when the app's airbrake initializer _also_ calls `Airbrake.configure`, some of the app's airbrake settings take effect (project_id, project_key, etc.), but the notifiers wouldn't get `reset` because they already exist (but lack some filters). Could that be what's going on?

**Comment by scream3** [comment]
I can confirm the issue. 
Now `Airbrake::NoticeNotifier#add_default_filters` is called before `initializers/airbrake.rb` is loaded.

**Comment by kyrylo** [comment]
Should be fixed in v4.2.4.
